File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Admin/ViajeDto.ts
export interface ViajeDto {
    id: number;
    driverName: string;
    passengers: string[];  // Lista de nombres de pasajeros
    origin: string;
    destination: string;
    startDate: Date;
    status: string;  // Puede ser PENDING, FINISHED, etc.
  }
  

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/admin/ViajeDto.java
package ps.com.viajeros.dtos.admin;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ViajeDto {
    private Long id;
    private String driverName;
    private List<String> passengers; // Lista de nombres de pasajeros
    private String origin;
    private String destination;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
    private LocalDateTime startDate;
    private String status; // PENDING, FINISHED, etc.

    // Getters y Setters
}
